keyboards/kyria/keymaps/gotham/joystick.c

```diff
@@ -0,0 +1,132 @@
+#if !defined JOYSTICK_CHANNEL_X || !defined JOYSTICK_CHANNEL_Y
+#    error "You must define JOYSTICK_CHANNEL_X and JOYSTICK_CHANNEL_Y in config.h, see keyboards\kyria\keymaps\gotham\README.md for more information."
+#endif
+
+#include "timer.h"
+#include "pointing_device.h"
+#include "LUFA/Drivers/Peripheral/ADC.h"
```

Comment:
Should be refactored to use <https://docs.qmk.fm/#/adc_driver>

Reply:
I can't believe I missed this in the docs. Thanks, I'll look into it!